src/BloomExe/Book/BookStorage.cs

```diff
@@ -656,9 +658,16 @@ private void UpdateSupportFiles()
 
 			//by default, this comes from the collection, but the book can select one, inlucing "null" to select the factory-supplied empty xmatter
 			var nameOfXMatterPack = _dom.GetMetaValue("xMatter", _collectionSettings.XMatterPackName);
-			var helper = new XMatterHelper(_dom, nameOfXMatterPack, _fileLocator);
-			UpdateIfNewer(Path.GetFileName(helper.PathToStyleSheetForPaperAndOrientation), helper.PathToStyleSheetForPaperAndOrientation);
 
+			try
+			{
+				var helper = new XMatterHelper(_dom, nameOfXMatterPack, _fileLocator);
+				UpdateIfNewer(Path.GetFileName(helper.PathToStyleSheetForPaperAndOrientation), helper.PathToStyleSheetForPaperAndOrientation);
+			}
+			catch (Exception error)
```

Comment:
Could we reasonably distinguish errors we should report like this (FileNotFound, IOException, maybe some to do with invalid XML) from more serious problems that are definitely bugs, like NullReference, KeyNotFound, OutOfMemory,...I know this is often hard and can live with this if you say so, but I (and MS guidelines) consider catching Exception (without differentiation) a code smell. I'm trying to resist doing it more than we already do.